exporting/json/metadata.yaml

```diff
@@ -0,0 +1,155 @@
+# yamllint disable rule:line-length
+---
+id: 'export-json'
+meta:
+  name: 'JSON'
+  link: 'https://cloud.google.com/pubsub'
+  categories:
+    - export
+  icon_filename: 'json.png'
+keywords:
+  - exporter
+  - json
+overview:
+  exporter_description: |
+    Use the JSON connector for the exporting engine to archive your agent's metrics to JSON document databases for long-term storage,
+    further analysis, or correlation with data from other sources
+  exporter_limitations: ''
+setup:
+  prerequisites:
+    list:
+      - title: ''
+        description: ''
+  configuration:
+    file:
+      name: 'exporting.conf'
+    options:
+      description: |
+        The following options can be defined for this exporter.
+      folding:
+        title: 'Config options'
+        enabled: true
+      list:
+        - name: 'enabled'
+          default_value: 'no'
+          description: 'Enables or disables an exporting connector instance (yes|no).'
+          required: true
+        - name: 'destination'
+          default_value: 'pubsub.googleapis.com'
+          description: 'Accepts a space separated list of hostnames, IPs (IPv4 and IPv6) and ports to connect to. Netdata will use the first available to send the metrics.'
+          required: true
+          detailed_description: |
+            The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+            - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+            - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+            - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used.
+
+            Example IPv4:
+              ```yaml
+              destination = 10.11.14.2:4242 10.11.14.3:4242 10.11.14.4:4242
```

Comment:
Our readme uses the following address:

```sh
localhost:5448
```

let us use it to avoid confusion with other exporters.

Reply:
These are generic for config options.